src/mod_websockets.erl

```diff
@@ -307,6 +319,11 @@ replace_stream_ns(Element, #ws_state{ open_tag = open }) ->
 replace_stream_ns(Element, _State) ->
     Element.
 
+get_parser_opts(Text, #ws_state{ max_stanza_size = infinity } = State) ->
+    [{max_child_size, 0} | get_parser_opts(Text)];
```

Comment:
do we need to set it to 0?

Reply:
No, not really - I just wanted it to be a) explicit b) identical to ejabberd_receiver's code. Do you want me to add a commit that removes this?

Reply:
I don't think it really matters, as it matches the default value assumed by `exml`.